src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -149,15 +148,13 @@ def create_webapp(cmd, resource_group_name, name, plan, runtime=None, startup_fi
             raise CLIError("usage error: --startup-file or --deployment-container-image-name or "
                            "--multicontainer-config-type and --multicontainer-config-file is "
                            "only appliable on linux webapp")
-        match = helper.resolve(runtime)
+        match = runtime.lower() in [s.lower() for s in RUNTIME_STACKS['windows']]
         if not match:
             raise CLIError("Runtime '{}' is not supported. Please invoke 'list-runtimes' to cross check".format(runtime))  # pylint: disable=line-too-long
-        match['setter'](cmd=cmd, stack=match, site_config=site_config)
         # Be consistent with portal: any windows webapp should have this even it doesn't have node in the stack
-        if not match['displayName'].startswith('node'):
```

Comment:
@panchagnula Why were we setting WEBSITE_NODE_DEFAULT_VERSION only if displayName didn't start with 'node'?

Reply:
:) Good question -> lets keep it that way for now since I don't want to break something - lets remove these in the next sprint to be safe after testing locallu.